doc/release-notes.md

```diff
@@ -1,11 +1,14 @@
-Dash Core version 0.16.1.1
-==========================
+Dash Core version 0.17
+======================
 
 Release is now available from:
 
   <https://www.dash.org/downloads/#wallets>
 
-This is a new hotfix release.
+This is a new major version release, bringing new features, various bugfixes
+and other improvements.
+
+This release is mandatory for all nodes.
```

Comment:
We should expound further, detailing that this release includes a BIP9 style hard fork that will begin signalling on July 1st, and that any nodes that do not upgrade by the time the HF is activated will diverge from the rest of the network

Reply:
bump?

Reply:
pls see 4d1c0b7